### ISSUE TITLE
Merge the two table of contents of the glossary into one

### ISSUE THREAD

**Issue by rontu85** [tightenco/onramp#96]
Currently there are two table of contents in the glossary. These two should be merged into one. As a result aria-hidden is not needed anymore.
